block.go

```diff
@@ -561,10 +554,17 @@ Outer:
 	pb.tombstones = stones
 	pb.meta.Stats.NumTombstones = pb.tombstones.Total()
 
-	if err := writeTombstoneFile(pb.logger, pb.dir, pb.tombstones); err != nil {
+	n, err := writeTombstoneFile(pb.logger, pb.dir, pb.tombstones)
```

Comment:
I think we might be able to avoid calculating size on write for both meta and tombstone if we just read both at the end. But this might be even more confusing, so not sure. Happy with this for now.

Reply:
yeah I think it will be confusing as well. will refactor if needed at some point.